<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>371</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="glLayout"/>
  </widget>
  <widget class="QDial" name="dialTheta">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>30</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dialPhi">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dialRotx">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dialRoty">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDial" name="dialRotz">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>150</y>
     <width>81</width>
     <height>101</height>
    </rect>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MainDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MainDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
